src/tests/functional/plugin/gpu/single_layer_tests/convolution.cpp

```diff
@@ -0,0 +1,125 @@
+        result << "PE" << CommonTestUtils::vec2str(padEnd) << "_";
+        result << "D=" << CommonTestUtils::vec2str(dilation) << "_";
+        result << "O=" << convOutChannels << "_";
+        result << "AP=" << padType << "_";
+        result << "netPRC=" << netType << "_";
+        result << "inPRC=" << inType << "_";
+        result << "outPRC=" << outType << "_";
+        result << "trgDev=" << targetDevice;
+
+        return result.str();
+    }
+
+protected:
+    void SetUp() override {
+        convSpecificParams convParams;
+        InputShape inputShape;
+        auto netType = ElementType::undefined;
+        std::tie(convParams, netType, inType, outType, inputShape, targetDevice) = this->GetParam();
+
+        init_input_shapes({inputShape});
+
+        ngraph::op::PadType padType;
+        InferenceEngine::SizeVector kernel, stride, dilation;
+        std::vector<ptrdiff_t> padBegin, padEnd;
+        size_t convOutChannels;
+        std::tie(kernel, stride, padBegin, padEnd, dilation, convOutChannels, padType) = convParams;
+
+        auto inputParams = ngraph::builder::makeDynamicParams(inType, inputDynamicShapes);
+        auto paramOuts = ngraph::helpers::convert2OutputVector(ngraph::helpers::castOps2Nodes<ngraph::op::Parameter>(inputParams));
+
+        auto convolutionNode = ngraph::builder::makeConvolution(paramOuts.front(), netType, kernel, stride, padBegin,
+                                                                padEnd, dilation, padType, convOutChannels);
+
+        ngraph::ResultVector results;
+        for (int i = 0; i < convolutionNode->get_output_size(); i++)
+                results.push_back(std::make_shared<ngraph::opset1::Result>(convolutionNode->output(i)));
+
+        function = std::make_shared<ngraph::Function>(results, inputParams, "Convolution");
+    }
+};
+
+TEST_P(ConvolutionLayerGPUTest, CompareWithRefs) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+    run();
+}
+
+namespace {
+INSTANTIATE_TEST_SUITE_P(smoke_ConvolutionLayerGPUTest_3D_tensor_basic, ConvolutionLayerGPUTest,
+        ::testing::Combine(
+                ::testing::Combine(
+                        ::testing::Values(SizeVector{3}),
+                        ::testing::Values(SizeVector{1}),
+                        ::testing::Values(std::vector<ptrdiff_t>{0}),
+                        ::testing::Values(std::vector<ptrdiff_t>{0}),
+                        ::testing::Values(SizeVector{1}),
+                        ::testing::Values(13),
+                        ::testing::Values(ngraph::op::PadType::SAME_UPPER)),
+                ::testing::Values(ElementType::f16),
+                ::testing::Values(ElementType::f16),
+                ::testing::Values(ElementType::undefined),
+                ::testing::Values(InputShape{{}, {{1, 13, 30}}}),
+                ::testing::Values<std::string>(CommonTestUtils::DEVICE_GPU)),
+                ConvolutionLayerGPUTest::getTestCaseName);
+}  // namespace
+
```

Comment:
Could you add a brief description of this test case?

Reply:
Added